package.json

```diff
@@ -46,7 +46,9 @@
       "packages/arb-bridge-eth",
       "packages/arb-provider-ethers",
```

Comment:
I think we're ready to remove the old provider package

Reply:
soon, but let's hold off on removing until we update the remaining scripts depencing on it (i.e., setup_validators_demo)